pallets/asset/src/lib.rs

```diff
@@ -453,25 +491,36 @@ decl_module! {
             let sender = ensure_signed(origin)?;
             let did = Context::current_identity_or::<Identity<T>>(&sender)?;
             Self::ensure_create_asset_parameters(&ticker, &name, total_supply)?;
-            let is_ticker_available_or_registered_to = Self::is_ticker_available_or_registered_to(&ticker, did);
-            ensure!(
-                is_ticker_available_or_registered_to != TickerRegistrationStatus::RegisteredByOther,
-                Error::<T>::TickerAlreadyRegistered
-            );
+
+            // Ensure its registered by DID or at least expired, thus available.
+            let available = match Self::is_ticker_available_or_registered_to(&ticker, did) {
+                TickerRegistrationStatus::RegisteredByOther => return Err(Error::<T>::TickerAlreadyRegistered.into()),
+                TickerRegistrationStatus::RegisteredByDid => false,
+                TickerRegistrationStatus::Available => true,
+            };
+
             if !divisible {
                 ensure!(total_supply % ONE_UNIT.into() == 0.into(), Error::<T>::InvalidTotalSupply);
             }
-            // Once all the checks are made, charge the protocol fee.
-            <<T as IdentityTrait>::ProtocolFee>::charge_fee(ProtocolOp::AssetCreateAsset)?;
+
+            // Once all the checks are made, charge the protocol fee if we should.
+            // Waive the fee iff classic ticker hasn't expired, and it was already created on classic.
+            if available || ClassicTickers::get(&ticker).filter(|r| r.is_created).is_none() {
+                <<T as IdentityTrait>::ProtocolFee>::charge_fee(ProtocolOp::AssetCreateAsset)?;
```

Comment:
Edge case: 
ACME is a token on Ethereum, owned by Alice
Alice does not claim ACME on Polymesh
Bob registers ACME ticker after the claim hold has expired
Bob creates ACME asset. As per the current logic, Bob will not need to pay the `AssetCreateAsset`. However, we actually want Bob to pay the fee.